django/applications/catmaid/static/js/WindowMaker.js

```diff
@@ -130,6 +130,15 @@ var WindowMaker = new function()
    * configuration.
    */
   var createWidget = function(instance) {
+    try {
+      CATMAID.throwOnInsufficientWebGlContexts(instance.MIN_WEBGL_CONTEXTS);
```

Comment:
That was intentional, because as you say an undefined does pass the test, but I can change it for clarity.